Microsoft.Toolkit.Uwp.UI.Controls/PullToRefreshListView/PullToRefreshListView.cs

```diff
@@ -33,6 +34,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     [TemplatePart(Name = PartIndicatorTransform, Type = typeof(CompositeTransform))]
     [TemplatePart(Name = PartDefaultIndicatorContent, Type = typeof(TextBlock))]
     [TemplatePart(Name = PullAndReleaseIndicatorContent, Type = typeof(ContentPresenter))]
+    [Obsolete("The PullToRefreshListView will be removed in a future major release. Please use the RefreshContainer control available in the 1803 version of Windows")]
```

Comment:
Is the sample already marked deprecated as well?

Reply:
I just noticed I updated most of the documentation, but forgot to add the badge for the menu and the disclaimer on top of the page in the sample app. I'll add them as soon as I can.

Reply:
Just added the badge and the deprecated message to the menu, also the deprecation warning on the page.